packages/quantic/force-app/main/default/lwc/quanticDidYouMean/quanticDidYouMean.js

```diff
@@ -38,6 +38,23 @@ export default class QuanticDidYouMean extends LightningElement {
    * @type {string}
    */
   @api engineId;
+  /**
+   * Whether to disable automatically applying corrections for queries that would otherwise return no results.
+   * When `disableAutomaticallyCorrectQuery` is `false`, the component automatically triggers a new query using the suggested term.
+   * When `disableAutomaticallyCorrectQuery` is `true`, the component returns the suggested term without triggering a new query.
+   * @api
+   * @type {boolean}
+   */
+  @api disableAutomaticallyCorrectQuery = false;
+  /**
+   * Defines which query correction system to use.
+   * `legacy`: Query correction is powered by the legacy index system. This system relies on an algorithm using solely the index content to compute the suggested terms.
+   * `next`: Query correction is powered by a machine learning system, requiring a valid query suggestion model configured in your Coveo environment to function properly. This system relies on machine learning algorithms to compute the suggested terms.
+   * The `legacy` system will send two requests to the API to get the suggestions, while the `next` system will send one request.
+   * @api
+   * @type {string} Possible values are `legacy` and `next`.
+   */
+  @api queryCorrectionMode = 'next';
```

Comment:
Here the Atomic component sets this to `next` by default, but we used to set it to `Legacy` in headless. Do we want to leave it to next?